include/swift/ABI/MetadataValues.h

```diff
@@ -19,6 +19,8 @@
 #ifndef SWIFT_ABI_METADATAVALUES_H
 #define SWIFT_ABI_METADATAVALUES_H
 
+// SWIFT_ENABLE_TENSORFLOW
+#include "swift/AST/AutoDiff.h"
```

Comment:
`#include "swift/AST/AutoDiff.h"` is included here to use `DifferentiabilityKind`, which may not be optimal.
An alternative is to define `DifferentiabilityKind` in this file and include this file in AutoDiff.h.

Reply:
No longer relevant after https://github.com/apple/swift/pull/25804/commits/3c1f0218dd14a87a54db31d58d97a2c5411e7297.
`ABI/MetadataValues.h` and `AST/AutoDiff.h` do not depend on each other, which is as things should be.